app/interactives/savings-calculator/page.tsx

```diff
@@ -312,7 +313,8 @@ export default function SavingsCalculator() {
                   <Input
                     id="current-balance"
                     type="number"
-                    value={currentBalance}
+                    value={currentBalance === 0 ? "" : currentBalance}
+                    placeholder="Current savings balance"
```

Comment:
The current balance needs to be allowed to be 0. When it's blank it's 0, so it works still, but it's not readily apparent to the user. Maybe the placeholder value could be 

```suggestion
                    placeholder="Current savings balance (leave blank for 0)"
```

Or there's some help text somewhere that says it?